### PR TITLE
CI: fix feather test

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

File: pandas/tests/io/test_common.py [hunk 1]
```diff
@@ -154,10 +154,14 @@ def test_read_non_existant(self, reader, module, error_class, fn_ext):
             fr"\[Errno 2\] File o directory non esistente: '.+does_not_exist\.{fn_ext}'"
         )
         msg8 = fr"Failed to open local file.+does_not_exist\.{fn_ext}.?, error: .*"
-
+        msg9 = (
+            # Encountered in feather 2020-02-07
+            fr"\[Errno 2\] Failed to open local file '.*does_not_exist\.{fn_ext}'. "
+            r"Detail: \[errno 2] 没有那个文件或目录"
+        )
```
Comment (jorisvandenbossche) [inline pandas/tests/io/test_common.py]: Or we can just stop testing the message here?

Comment (jbrockmendel) [reply pandas/tests/io/test_common.py]: i would be on board for that, cc @MomIsBestFriend looks like there is a new external_error_raised; is that for roughly this purpose?

Comment (MomIsBestFriend) [reply pandas/tests/io/test_common.py]: > i would be on board for that, cc @MomIsBestFriend looks like there is a new external_error_raised; is that for roughly this purpose?

@jbrockmendel Yes, exactly!

**Comment by jbrockmendel** [comment]
updated to use external_error_raised.  Do we want to check anything about the message being reasonable?

**Comment by MomIsBestFriend** [comment]
> updated to use external_error_raised. Do we want to check anything about the message being reasonable?

I don't think we do, as this is an external error message.
But I would like to hear others as well.

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @jbrockmendel yea I don't think we need to test for 8 different messages!

**Comment by WillAyd** [comment]
Hmm I guess that helper isn't available on the 1.0.x branch

**Comment by jreback** [comment]
why exactly was this merged @WillAyd @jbrockmendel @jorisvandenbossche  was this fixing a CI issue?

it is not about testing for the error messages, but the fact that the error messages are *different* that is extremely troubling. This is de-facto removing legitimate tests.

**Comment by WillAyd** [comment]
Yea looks to have failed CI

**Comment by WillAyd** [comment]
I think alternately would have to add a 9th error message to test for with current design; maybe should break up and test error message specifically by reader?

**Comment by simonjayhawkins** [comment]
msg4 doesn't look like an external message. should external_error_raised fail this? @MomIsBestFriend

**Comment by simonjayhawkins** [comment]
not at a computer to check though. so could be external. just looks like one of ours.

**Comment by jorisvandenbossche** [comment]
> msg4 doesn't look like an external message. 

Yes, I think that is the only one raised by ourselves (it's "path_or_buf needs to be a string file path or file-like"). But it's actually also a wrong error message we were testing .. as what is passed *is* a string file path)

**Comment by MomIsBestFriend** [comment]
> msg4 doesn't look like an external message. should external_error_raised fail this? @MomIsBestFriend

@simonjayhawkins ```tm.external_error_raised``` is an alias for ```pytest.raises(<An exception class>, match=None)```, it does not check for whether or not this is an actual external error message.

**Pull request merged**